src/d3d11/d3d11_device.cpp

```diff
@@ -2531,7 +2531,8 @@ namespace dxvk {
       return S_OK;
     }
     
-    if (riid == __uuidof(IDXGIVkInteropDevice)) {
+    if (riid == __uuidof(IDXGIVkInteropDevice)
+     || riid == __uuidof(IDXGIVkInteropDevice2)) {
```

Comment:
To keep this consistent with Microsoft's weird D3D naming schemes, this should be `IDXGIVkInteropDevice1`, and `GetSubmissionQueue1`, respectively.